teacher.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
+  def initialize(name, age, specialization, parent_permission)
+    super(name, age, parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] the default values for name and parent_permission should be included here as well.

Reply:
@mikethreels Thank you!